### ISSUE TITLE
Follow Threads Opt-in

### ISSUE THREAD

**Issue by pfefferminzz** [ludumdare/ludumdare#1468]
Please make NOT following a thread the default state. Following should be opt-in.

**Comment by povrazor** [comment]
If focus was limited to replies only (currently you can't reply to comments) it might not be so annoying.

Alternatively it was suggested that auto-subscribing could be a user preference, but I think the core problem is actually that because you can't reply to people, we have to subscribe to the entire thread.

**Comment by pfefferminzz** [comment]
Mentions of @username creating a notice for that user would be so nice, you could reply to comments individually.

**Comment by povrazor** [comment]
Threading is supported behind the scenes, but we haven't yet finished the UI for it (other things were higher priority).

**Comment by sgstair** [comment]
@username mentions do always produce a notification for the mentioned user, Are you observing otherwise?

**Comment by pfefferminzz** [comment]
No, i just did not know that it works already (as nobody mentioned me outside of my game page).
I used @username to answer comments all the time. It's great that those users are getting a notice!

**Comment by blinry** [comment]
Seconding @pfefferminzz's original request: Please don't auto-subscribe to a game when commenting. This seems like an anti-feature, which you should disable right now. Please :) It's annoying to have to manually unsubscribe to each game, so that the notifications aren't completely cluttered.

**Comment by povrazor** [comment]
We don't yet have a page that tells you what you've rated (though there is an API call https://api.ldjam.com/vx/grade/getgames). At this very moment subscribing does act like a poor-mans "what have I rated", though this is definitley not a long term solution.

**Comment by blinry** [comment]
I see, but in my eyes, this is a suboptimal workaround. People can use the userpage on feedback.ld.intricati.com if they want to see which games they have commented on, until this functionality is on ldjam.com.

Right now, the workaround causes spammy, unwanted notifications, and I'd strongly advocate for making the subscriptions opt-in as soon as possible.

**Comment by local-minimum** [comment]
I implemented a notifications filter on #1494 that will not undo the notifications but will put the pure comments (not on your stuff and not a mention) as filtered out. For now this will only be active on the notifications feed (given that it is accepted). But it could also easily be extended to the notifications drop-down if we stored the user's settings in a cookie or similar.

**Comment by sgstair** [comment]
The other question that I can't answer here, is for how many people is the current default behavior the expected and desired behavior - I know at least it is working how I would prefer it to work. This just really highlights the variety of settings that will be useful for customizing this, I think.

**Comment by pfefferminzz** [comment]
Maybe there should be a notice for someone giving you Karma with a link to your post. If you leave a quality comment on the games you play the dev will likely give you a heart, so many games you played will pop up in your messages but likely only once or twice and not for every comment in that game thread.

**Comment by local-minimum** [comment]
My thought is that this would be better handled through user settings than per thread basis. Though I can see the point in filtering out comments in general but really wanting to know about everything on some post even if you didn't write the original post. 

So my vote would be now that the filter is live, to aim for the filter-settings to stick and be respected by the drop-down too and then not bother too much about if opt-in is default. Only annoying thing is we might want to load more notifications on the feed at a time to compensate for that you probably will filter out the majority of them.

**Comment by mikekasprzak** [comment]
Noting this is related to #701, but its own issue.

**Comment by local-minimum** [comment]
I haven't heard any new complaints since filters were introduced so I suggest closing. Comment threading should be its own issue
